bin/ream/src/main.rs

```diff
@@ -144,9 +145,17 @@ pub async fn run_lean_node(config: LeanNodeConfig, executor: ReamExecutor) {
     // TODO 1: Load keystores from the config.
     // TODO 2: Add RPC service for lean node.
     let chain_service = LeanChainService::new(lean_chain.clone(), chain_receiver).await;
+    let gossipsub_config = LeanGossipsubConfig::default();
```

Comment:
why did you un-inline this? It doesn't look like this variable is used anywhere and it isn't complex so it should just be inlined again